src/style.css

```
@@ -1 +1,41 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(-n),
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 1),
+.stars--3 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
```

```suggestion
.stars--3 :nth-child(-n + 3),
```
you can simplify that, and do the same for other